gallery/how_to/work_with_microtvm/micro_tvmc.sh

```diff
@@ -87,7 +87,7 @@ tvmc micro --help
 # For this tutorial we will be using the model in tflite format.
 #
 # bash
-wget https://github.com/tensorflow/tflite-micro/raw/main/tensorflow/lite/micro/examples/magic_wand/magic_wand.tflite
+wget https://github.com/tensorflow/tflite-micro/raw/1b68490dcfd087c308a1f8d14012ab1cb39c9ac0/tensorflow/lite/micro/examples/magic_wand/magic_wand.tflite
```

Comment:
Thanks for the fix 😄 I wonder if we should update the example to use one of the currently listed upstream examples e.g. [micro speech](https://github.com/tensorflow/tflite-micro/blob/main/tensorflow/lite/micro/examples/micro_speech/micro_speech.tflite).

Obviously this would be prone to the same error we have just seen if that model ever got removed, however it does avoid us having a reference to a random commit in this example.

Reply:
I'll update the example to use [micro_speech](https://github.com/tensorflow/tflite-micro/blob/main/tensorflow/lite/micro/examples/micro_speech/micro_speech.tflite) as it's used in other examples.

Reply:
Awesome, thank you 😄 

I think we will also need to update references to `magic wand` [here](https://github.com/apache/tvm/blob/main/gallery/how_to/work_with_microtvm/micro_tvmc.sh#L84), [here](https://github.com/apache/tvm/blob/main/gallery/how_to/work_with_microtvm/micro_tvmc.sh#L103) and [here](https://github.com/apache/tvm/blob/main/gallery/how_to/work_with_microtvm/micro_tvmc.sh#L106).